BioscoopSysteemAPI/BioscoopSysteemAPI/Migrations/20230228204203_20230228_SecondMigration.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
```

Comment:
Waar is dit bestand voor?

Reply:
Geen idee tbh. Ik heb niks met de backend gedaan. Dus ik denk dat die weg kan?

Reply:
Removed.